jdi-light-material-ui-tests/src/main/java/io/github/com/pages/utils/TransitionPage.java

```diff
@@ -0,0 +1,22 @@
+package io.github.com.pages.utils;
+
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.XPath;
+import com.epam.jdi.light.ui.html.elements.common.Checkbox;
+
+import java.util.List;
+
+public class TransitionPage extends WebPage {
+
+    @UI("//*[contains(@class,'MuiPaper-elevation4 MuiPaper-rounded')]")
```

Comment:
please use more compact and universal locator, because other class values can be changed in future. For example:
//*[contains(@class,'MuiPaper-root')]